style-guide/src/styles/index.module.css

```diff
@@ -1,8 +1,51 @@
-.body {
-  color: palegreen;
+@value spacing-medium from 'otkit-spacing/token.cssmodules.css';
+@value color-primary, color-gray-primary from 'otkit-colors/token.cssmodules.css';
+@value font-weight-medium from 'otkit-typography/token.cssmodules.css';
+
+/* local variables */
+@value width-two-column: 992px;
+@value width-nav: 320px;
+@value width-main: 640px;
+
+body {
+	font-family: 'BrandonText', -apple-system, BlinkMacSystemFont, 'Segoe UI',
```

Comment:
we probably actually 100% wanna see the actual font, not the fallbacks, so I don't know if the full stack here is useful or bad. Actually, we probably want to have a way of seeing the fallbacks live, so we know they look like when the font doesn't load. Another feature for later.